pkgconfig/JoinPaths.cmake

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+# This module provides function for joining paths
+# known from most languages
+#
+# Copyright 2020 Jan Tojnar
```

Comment:
We very much prefer all contributions to be `Copyright The Mbed TLS Contributors` @jtojnar is this OK with you?

Reply:
Sure.

Reply:
Done, thanks @jtojnar